app/controllers/movies_controller.rb

```diff
@@ -22,9 +32,9 @@ def keywords
     json1 = JSON.parse(page1.body, symbolize_names: true)
     json2 = JSON.parse(page2.body, symbolize_names: true)
 
-    movies = json1[:results] + json2[:results]
+    make_movies(json1, json2)
```

Comment:
This is the line where what we used to have deviates pretty wildly, BUT I think most of what you're doing is above this in this method, which means the `make_movies` method would just have to be refactored to take in 1 argument instead of 2.  It should work pretty well I think!  And this spares us from having to store a billion things from the API.

Reply:
This does clash with what I've made but if we can maybe look at it again once Ive finished with my movies_page branch and see what works better.

Reply:
I think this should now be resolved.  Let me know if it's not!